internal/common/disasters/copy.go

```diff
@@ -0,0 +1,13 @@
+package disasters
+
+// Copy returns a deep copy of Environment.
+func (e Environment) Copy() Environment {
```

Comment:
Ahh okay - misunderstood how that worked. That's a lot cleaner.

Would it have worked the other way btw (despite being not as nice)? The other way being using an independent function, not a Copy() method

Reply:
Yep, it could! But I feel having its own `Copy` is nicer. Do you want to take over and export some fields to make it output more? Just push commits to `infra-core-copy`

Reply:
Yeah agree - def is nicer. Okay sure, will do.